types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -336,6 +336,33 @@ declare namespace GoogleAppsScript {
             getSpec(): DataSourceSpec;
             updateSpec(spec: DataSourceSpec): DataSource;
         }
+        /**
+         * Access and modify a data source column.
+         *
+         * Only use this class with data that's connected to a database.
+         */
+        interface DataSourceColumn {
+            getDataSource(): DataSource;
+            getFormula(): string;
+            getName(): string;
+            hasArrayDependency(): boolean;
+            isCalculatedColumn(): boolean;
+            remove(): void;
+            setFormula(formula: string): DataSourceColumn;
+            setName(name: string): DataSourceColumn;
+        }
+        /**
+         * Access and modify an existing data source sheet filter. To create a new data source sheet filter, use DataSourceSheet.addFilter(columnName, filterCriteria).
+         *
+         * Only use this class with data that's connected to a database.
+         */
+        interface DataSourceFilter {
```

Comment:
Rename to DataSourceSheetFilter and move after DataSourceSheet.
https://developers.google.com/apps-script/reference/spreadsheet/data-source-sheet-filter